src/trades.py

```diff
@@ -0,0 +1,74 @@
+"""
+Definition of trades, orders, and executions.
```

Comment:
NIT: File name should perhaps be models.py then (if you have more than one model). In the future the models may expand into more than just these three. You may want to create a directory `models` and have one file per model (`model/trade.py`, `model/order.py`, etc...)